src/action.c

```diff
@@ -951,6 +953,12 @@ actions_run(struct view *activator, struct server *server,
 				}
 			}
 			break;
+		case ACTION_TYPE_TOGGLE_TEARING:
+			if (view) {
+				view->tearing_hint = !view->tearing_hint;
+			}
+			wlr_log(WLR_DEBUG, "tearing: %d", view->tearing_hint);
```

Comment:
This should be inside the `if (view)`.

Reply:
Right, also maybe i should make it say enabled/disabled instead of 0/1.